addon/mixins/scoped-query-cache.js

```diff
@@ -30,29 +42,19 @@ export default Ember.Mixin.create({
           formatCacheKey(query) :
           query;
 
-    assert('queryCache needs to be an object', Ember.typeOf(queryCache) === 'object');
+    assert('queryCache needs to be an object', typeOf(queryCache) === 'object');
```

Comment:
let's remove the parenthesis on line 41 so that the native `typeof` and the destructured `typeOf` are a tad more different visually.

`Ember.typeOf` is kinda crappy (slow and stuff) although this being a development assertion it is likely fine, but maybe a tad confusing?


is the use of `Ember.typeOf` here essentially the following?

```js
typeof queryCache === 'object' && queryCache !== null && !Array.isArray(queryCache)
```

Would it be clearer to write it out in long-form?